app.js

```diff
@@ -1,66 +1,77 @@
-// Copyright 2016, EMC, Inc.
-/* jshint node: true */
-
-"use strict";
-
-var di = require('di');
-
-module.exports = Runner;
-
-di.annotate(Runner, new di.Provide('app'));
-di.annotate(Runner, new di.Inject(
-    'Http.Server',
-    'Services.Configuration',
-    'fileService',
-    'Logger',
-    'Promise'
-)
-);
-function Runner(
-    HttpService,
-    configuration,
-    fileService,
-    Logger,
-    Promise
-) {
-    var logger = Logger.initialize("app");
-
-    var services = [];
-
-    var defaultEndpoints = [
-        {
-            "address": "0.0.0.0",
-            "port": 8080,
-            "routers": "northbound"
-        },
-        {
-            "address": "0.0.0.0",
-            "port": 9080,
-            "routers": "southbound"
-        }
-    ];
-
-    function start() {
-        return Promise.resolve()
-            .then(function () {
-                logger.info('debug, in service start');
-                var endpoints = configuration.get('httpEndpoints', defaultEndpoints);
-                services = Promise.map(endpoints, function (endpoint) {
-                    return new HttpService(endpoint);
-                })
-
-                return services;
-            });
-    }
-
-    function stop() {
-        return Promise.map(services, function (service) {
-            return service.stop();
-        })
-    }
-
-    return {
-        start: start,
-        stop: stop
-    };
-}
+// Copyright 2016, EMC, Inc.
+/* jshint node: true */
+
+"use strict";
+
+var di = require('di');
+
+module.exports = Runner;
+
+di.annotate(Runner, new di.Provide('app'));
+di.annotate(Runner, new di.Inject(
+    'Http.Server',
+    'Services.Configuration',
+    'Logger',
+    'fs-operation',
```

Comment:
A little suggestion, Why not keep the name in the same style with others, such as "Fs.Operation"?